doc/tools/docgram/fullGrammar

```diff
@@ -328,6 +328,7 @@ closed_binder: [
 | "(" name ":" lconstr ")"
 | "(" name ":=" lconstr ")"
 | "(" name ":" lconstr ":=" lconstr ")"
+| "(" name ":" lconstr "|" lconstr ")"
```

Comment:
Shouldn't this production appear in the documentation somewhere

Reply:
it does appear, a bit above the paragraph I inserted in this PR